molgenis-core-ui/src/main/javascript/modules/react-components/EntitySelectBox.js

```diff
@@ -1,16 +1,22 @@
-/* global _: false, React: false, molgenis: true */
-(function(_, React, molgenis) {
-	"use strict";
-	
+import React from "react";
+import DeepPureRenderMixin from "./mixin/DeepPureRenderMixin";
+import EntityLoaderMixin from "./mixin/EntityLoaderMixin";
+import ReactLayeredComponentMixin from "./mixin/ReactLayeredComponentMixin";
+import RestClient from "rest-client/RestClientV1";
```

Comment:
We made rest-client into a separate module. It's also used on pages without react.